frappe_pwa/frappe_pwa/doctype/web_app_manifest/test_web_app_manifest.py

```diff
@@ -6,5 +6,13 @@
 import frappe
 import unittest
 
+from frappe_pwa.frappe_pwa.doctype.web_app_manifest.web_app_manifest import configure_pwa
+
+
 class TestWebAppManifest(unittest.TestCase):
-	pass
+    def test_configure_pwa(self):
+        configure_pwa()
+        ws = frappe.get_doc('Website Settings')
+        self.assertIn('<link href="/assets/frappe_pwa/manifest.json" rel="manifest">', ws.head_html)
```

Comment:
This obviously is wrong given the fact the address was fixed in `web_app_manifest.py `...

Just check `rel="manifest"` is in the header.